### PR TITLE
[s] Hellway: One of the maps of all time

### PR DESCRIPTION
**Pull request by Zonespace27**
TM only for the love of god

### CODE REVIEW

**Comment by Zonespace27** [comment]
I can TM closed PRs

**Comment by iloveloopers** [comment]
this appeared on the discord, you failed.

File: code/game/machinery/computer/computer.dm [hunk 1]
```diff
@@ -140,3 +140,60 @@
 /obj/structure/machinery/computer/fixer/New()
 	all_configs = config
 	..()
+
+/obj/structure/red_terminal
```
Comment (Huffie56) [inline code/game/machinery/computer/computer.dm:144]: if it's not using parent /obj/structure/machinery/computer
it should probably be in it's own file.

Comment (Zonespace27) [reply code/game/machinery/computer/computer.dm:144]: for one-off event PRs, i don't really care

**Pull request closed without merging**